fix(navigation-primary): hamburger standalones toggle state (#2462)
RedHat-UX/red-hat-design-system: .changeset/heavy-hornets-throw.md

---
"@rhds/elements": patch
---

> `<rh-navigation-primary>`: corrected dropdown toggle state managment, improved accessibility with keyboard navigation

Reviewer: ```suggestion
`<rh-navigation-primary>`: improved keyboard accessibility of the dropdown toggle
```
